core/queue/lua/done.lua

```diff
@@ -1,9 +1,10 @@
--- KEYS: [QueueName] [TaskGroup]
+local activeSetKey = KEYS[1]
+local ownerID = ARGV[1]
```

Comment:
I do want to do away with mention of orgs and make this code completely generic and live in redisx.. "owner" seems more intuitive than "group" in the context of fair task processing.